servatrice/src/servatrice.cpp

```diff
@@ -74,11 +74,11 @@ Servatrice_GameServer::~Servatrice_GameServer()
     }
 }
 
-void Servatrice_GameServer::incomingConnection(qintptr socketDescriptor)
+void Servatrice_GameServer::incomingConnection(const int &socketDescriptor)
```

Comment:
The base Qt class is declaring a `qintptr` argument (http://doc.qt.io/qt-5/qtcpserver.html#incomingConnection), i see no reason to change this.

Reply:
I didn't change this, which was weird... Clion didn't like the qintptr :P Will readdress this